scio-jdbc/src/main/scala/com/spotify/scio/jdbc/syntax/SCollectionSyntax.scala

```diff
@@ -30,6 +31,32 @@ final class JdbcScioContextOps(private val self: ScioContext) extends AnyVal {
   /** Get an SCollection for a JDBC query. */
   def jdbcSelect[T: ClassTag: Coder](readOptions: JdbcReadOptions[T]): SCollection[T] =
     self.read(JdbcSelect(readOptions))
+
+  /**
+   * Sharded JDBC read from a table or materialized view.
+   * @param readOptions The following paramters in the options class could be specified:
+   *
+   *                    shardColumn: the column to shard by. Must be
+   *                    of integer/long type ideally with evenly distributed values.
+   *
+   *                    numShards: number of shards to split the table into for reading.
+   *                    There is no guarantee that Beam will actually execute reads in parallel.
+   *                    It is up to Beam auto scaler to decide the level of parallelism to use
+   *                    (number of workers and threads per worker). But the behavior could be
+   *                    controlled with maxNumWorkers and numberOfWorkerHarnessThreads parameters
+   *                    (see more details about these parameters here). Defaults to 4.
+   *
+   *                    tableName: name of a table or materialized view to read from
+   *
+   *                    fetchSize: number of records to read from the
+   *                    JDBC source per one call to a database. Default value is 100,000. Set to
+   *                    -1 to make it unbounded.
+   */
+  def jdbcShardedSelect[T: ClassTag: Coder, S](
```

Comment:
`ClassTag` not needed? outside of this PR, but I see that the previous method also defines it ... it shouldn't be needed as well.

Reply:
Indeed, it isn't needed. I blindly copy pasted it from the `jdbcSelect`. Actually, `jdbcSelect` doesn't need it as well. Do you think I can safely remove it from there also (don't want to break anything )?